mochify/lib/run.js

```diff
@@ -8,15 +8,15 @@ const { mochaEventAdapter } = require('./mocha-event-adapter');
 
 exports.run = run;
 
-function run(driver, mocha_runner, script) {
+async function run(driver, mocha_runner, script) {
   let mapStack = null;
   const source_map = fromSource(script);
   if (source_map) {
     mapStack = stackMapper(source_map.toObject());
     script = removeComments(script);
   }
 
-  injectScript(driver, script);
+  await injectScript(driver, script);
```

Comment:
This is a drive by fix/improvement: as this method is async, we'd currently start polling for events before the script has been injected in the first place, which does not hurt in practice, but can be confusing when debugging.